spec/models/resource_category_spec.rb

```diff
@@ -44,4 +44,43 @@
 
     end
 
+    describe "validations" do 
```

Comment:
Ruby uses 2 space indentation

Reply:
Alright, I'll change the indentation. Might have to do it after class